### DISCUSSION TITLE
One autor top menu link

### DISCUSSION THREAD

**Discussion by kornevplus** [PxlSyl/tailwind-nextjs-starter-blog-i18n#16] [General]
Let's say I have only one author. But in the top menu, the **About** button still works as a drop-down menu. Is it possible to make it work as a simple link? Same as in the original template.
I understand that `AuthorsMenu` is responsible for this, but if you remove it, the link disappears completely.

**Comment by kornevplus** [comment]
I found the solution. Just go to `sitemetadata.js` and set `multiauthors` to `false`

**Comment by PxlSyl** [comment]
Hi! Sorry for the late. Yes, it's quite straightforward. The demo (and demo code) is set up with multiauthors = true, but that's for showing the capabilities of the template. In any way, you should feel free to remove or modify the unused features for your own website, or just set up multiauthors to false (as you did)
